src/screens/AnthropometricScreen.js

```diff
@@ -0,0 +1,170 @@
+import React, {useState, useEffect} from 'react';
+import { StyleSheet, View, Text, AsyncStorage, Button, TextInput, ScrollView } from 'react-native';
+import * as ThemeConstants from '../common/Themes';
+import Input from '../components/Input';
```

Comment:
Arrange imports alphabetically, according to folder. (i.e. ActivityInput comes before Input)